### DISCUSSION TITLE
基本收入 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#460] [General]
# 基本收入 - Miao Yu | 于淼

https://yufree.cn/cn/2024/03/26/ubi/

**Comment by hongtaoh** [comment]
很深度的思考。我在想的一个问题是，世界上所有人实现财富自由有没有可能？假设可能，所有人都到了自己设想的财富自由阶段，这个世界会变得更好吗？

**Reply by yufree** [reply]
财富自由不大可能，主要每个人定义不一样。大多数人的想象力只够设想自己当前收入翻倍的状态，更多时就开始脱离实际烙大饼了。假设一个人工作35年，每年工资排除通胀平均上浮2%，平平稳稳不折腾到退休前工资恰好是刚入职时的一倍。也就是说，只要问一下现在快退休的人是否满意当前经济状态就知道答案了。我的观察是国内绝大部分都很满意，因为当前退休的人起点工资最低只有两位数，享受了大环境进步的成果，但显然这个问题问日本人更合适，毕竟过去35年他们大环境基本没啥变化。至于说世界会不会变好，也取决于个人感受，很多卷人感受世界变好要把同行都踩在脚下才行，这种情况他们不到顶端永远不会认为世界变好了。换句话说，世界是否变好是个主观问题，因为每个人三观有差异，更多时候是在一个场景里感受到进步，另一个场景里感受到倒退。

**Comment by tcgriffith** [comment]
看过包括翟东升老师在内的不少关于基本收入的讨论，曾经我也颇感兴趣。但目睹了围追堵截之下华为搓出mate60， ，昔日西方工业皇冠上的明珠一个个被拿下(就差4090了！) 另一边又看到北美甜甜圈在急诊话疗一下的5670刀乐账单，我有一点不一样的看法。

就本文的UBI来说，感觉制度设计的理论再精巧完美，也顶不住米帝聊两句话就开五千刀的医疗账单的医疗体系啊。 

所以对米国的城市来说不需要UBI，印钱发钱不会增加商品和服务的供给，另一边还空增通胀。

中国的城市也不需要UBI。 二十年前桑塔纳都高攀不起，现在都比亚迪秦王扫六合了。有没有UBI那798的秦还不是分分钟全款拿下。

总之， 消除贫富差距这个目标太宏大也太复杂了，只靠UBI解决不了

**Reply by yufree** [reply]
不矛盾，贫富差距每个经济体或者说城市经济体都要面对与解决，否则无法维持长期繁荣稳定，跟哪个国家无关，而是任何国家或城市都存在，甚至越是技术发达，这个问题引发的矛盾就越显著。而且也不是说我对UBI有啥喜好，事实上很多行业或圈层在权力或技术门槛垄断下已经实现了某种程度的基本收入，例如富人仅通过利息收入就可以不增加产出条件下正常生活，UBI只是一种打补丁方式来缓解。

你提到的是技术进步后的成本降低，问题是这类技术普惠性有多少？有多少人需要mate60与秦？现在的尖端技术受益者更多是人数相对少的技术精英，更多人就业可能都有问题。不是说生产率上去了所有人都富了，而是生产率上去了很多行业不需要低端劳动密集型行业了，这些人没有UBI是可能饿死的。美国一样有那种一天工作十几小时拿最低工资的人天天在温饱线挣扎，国内同样有吃瓦片的大爷，普惠技术不足但生产力发达，后果就是贫富差距拉大，经济体内部割裂感严重，会出现大动荡或对外输出战争。

另外，印钱发钱对美国来说就是可以凭空增加商品与服务供给，最多就是多点数字化的债务，但其他国家玩这一手别的经济体不认这个主权债。或者说腾讯的q币如果可以直接无限制买到实物，那么一定是大多数人认定q币有一般等价物属性，理论上腾讯可以用q币发工资，只要腾讯不倒q币就可以继续流通换来外部商品与服务，这个通胀只要构建在腾讯不倒信仰下还是可维持的。

至于美国医疗，这个问题太复杂，可理解为医药利益共同体为了维护内部职业高薪特权发展出的产物，他们需要所有人认可医保来维持，不加入医保自然就惩罚你。可以类比国内烟草行业的世袭现状，本质就是对烟民的压榨，不抽烟他们确实也收割不到，一抽烟就吃一辈子利润。这跟UBI没啥关系，倒是跟行业内基本收入有点联系，但这种行业大都只能通过艾滋病渠道传播，普通人进入门槛极高，但内部已经实现UBI了。

我没说UBI能完美解决，只是说设计良好的UBI可以一定程度缓解当前的问题。另外贫富差距也不涉及国家竞争，在我看来国内技术进步与美国医疗保险制度都一定程度是全球范围内贫富差距的原因或结果，都无法正面应对贫富差距问题本身。

**Reply by tcgriffith** [reply]
你提到的几个观点我都部分认同，不过感觉前一个回复太简单了，下面我尝试重新组织语言阐述一下我的观点。

1. UBI尝试解决的问题是什么

”贫富差距每个经济体或者说城市经济体都要面对与解决，否则无法维持长期繁荣稳定“。

这一点不认同，我觉得贫富差距并不是一个需要解决的问题，对一个经济体维持长期繁荣稳定而言，解决贫富差距即不充分，也不必要。我觉得从普通大众的视角看，可能更关注一个城市经济体的中位数，或者底层10%能获得什么样的生活水平。但这个生活水平中，包括衣食住行，医疗教育安全娱乐，与经济相关的只是其中的一部分。 中位数水平代表了大部分人的生活状态如何， 底层10%水平代表了弱势群体能否兜得住。

我觉得主要的问题还是生产力发展，农业，工业，制造业的生产效率提高，**无法创造更多的工作岗位**。而UBI 给所有的人发钱，可以让没有工作的人也拥有被动收入，从而参与到基本的经济循环之中来。 本质还是一种底层10%的兜底性工具，对中位数作用有限，这个群体衣食住行基本开销没问题，房车等大项支出，UBI也帮不上多少。

2. UBI的局限性

如果想要解决的问题是就业，那相比UBI兜底。我觉得一般人更直接的想法是去创造就业岗位。 事实上国内一些发展旅游经济，生态农业的导向颇有这种感觉。服务业相比制造业吸纳就业人口的潜力还是蛮大的，毕竟剃头师傅再怎么努力也不可能剃八百个头。

另一方面，在高薪岗位有限且在减少，中位数人群的工资收入增长缓慢的现状下，我们发展科技的也是有不输UBI的意义的。将原来只能满足少数人需求的中高端产品的价格打下来，提高中位数人群生活水平也可以实现普惠。

> 有多少人需要mate60与秦

有，还不少。比亚迪把原来合资车中高配(20~30万)才有的配置搞到7.98万，按2023年人均可支配收入算，一个三口之家一年就能全款开上不错的车。798秦是国内工业和科技发展抬升中位数人群的生活水平的直接体现。

传统汽车强国就没那么幸运了，大众id3在德国卖四万欧，在国内卖1.5万欧还没啥销量，技术水平落后带来的结果是德国吹的工业4.0自己都不信。 日本车企在国内销量也跌跌不休，三菱都干退市了。


> 生产率上去了很多行业不需要低端劳动密集型行业了，这些人没有UBI是可能饿死的

各种社会都多多少少有兜底制度，UBI不干，有的是别人干！ （叉腰

**Reply by yufree** [reply]
首先，我们都认同生产力进一步发展无法创造更多工作岗位的情况，那么贫富差距作为一种现象只会越来越显著，你所关注的中位数会不断左移。都不用说未来，现在美国、日本等发达经济体的中产人数都是在[收缩](https://www.imf.org/zh/Publications/WEO/Issues/2023/10/10/world-economic-outlook-october-2023)的，相比阶层跃升，中产返贫的概率更高，因此进行制度兜底存在必要性。
其次，拿工资而非利息的人口历史上是按照三个产业进行转移的，工业革命把农业人口逼入工业流水线，高新技术把工业人口逼入服务业。整体上社会需求从物质产能不足逐渐进入物质产能过剩，因此只能转到精神需求无上限的服务业容纳就业，让价值交换更多发生在人与人之间而不是人与物之间，最后承担就业的确实就是服务业。不过作为下岗职工的二代，我深知这个转型对曾经的产业工人并不简单，大多数产业工人是没有这个思想跟技能准备的，后面自动化AI浪潮过来很多伪服务业（只会走流程的办公室白领与文职）中产返贫后带来的动荡也是可以预期的，UBI至少能预防点转型期的阵痛。站在旁观者视角，我们可以说不就是换个职业，但作为当事人，这并不容易。
再次，科技是第一生产力这点毋庸置疑，但现在很多高端技术例如精准医疗与保险定价根本就不是为了普惠设计的，甚至技术本身就依赖信息不对等与垄断来实现，例如香港金融业与台湾芯片行业都是产值大户，但其他承接大多数就业人口的行业只能喝汤或陪衬，吸引不到人才来发展民生行业。当然，我承认这个UBI也解决不了，只是想说技术趋势不一定是普惠。
最后，我知道秦便宜，五菱宏光更便宜，但这时候不要算国家人均啊，按城市统计，361个地级市里只有135个人均能达到秦，一多半也达不到这个标准，看中位数估计这个数还得打折一半，也就五分之一的人能用一年不吃不喝收入买个秦，前景是光明的，道路是曲折的。
善战者无赫赫之功，兜底制度最好要有前瞻性，等出了问题打补丁很多时候选项就少了。
